_data/constants.yml

```diff
@@ -0,0 +1,8 @@
+raft_election_timeout: 2s
+raft_proposal_timeout: 2-4s
+node_heartbeat_interval: 3s
+lease_interval: 6s
+lease_acquisition_timeout: 4s
+cockroach_network_timeout: 2s
+cockroach_network_connection_timeout: 4s
+cockroach_raft_election_timeout_ticks: 10
```

Comment:
In cockroachdb/cockroach#98584, the tick interval `COCKROACH_RAFT_TICK_INTERVAL` was changed from 200 ms to 500 ms, so the Raft election timeout was reduced from 10 ticks to 4 ticks (both are equivalent to 2 seconds in wall time).